com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UNetTransport.cs

```diff
@@ -79,6 +80,9 @@ protected void LateUpdate()
         {
             if (UnityEngine.Networking.NetworkTransport.IsStarted && MessageSendMode == SendMode.Queued)
             {
+#if UNITY_WEBGL
```

Comment:
What about throwing an exception?  Otherwise looks like it will just throw an error and then do nothing.

Reply:
Yeah, since we are following the pattern of throwing exceptions...but the weird part of this is that it will be in a browser... so haven't done a bunch of that...but it could be that the best you can do under that scenario is to just log the issue.  Really, we need to have a "internal MLAPI dialog box" that can display critical errors for users...so we know they will see it no matter what.

Reply:
We can't. It literately doesn't compile because those methods gets stripped for WebGL.